tests/attr/test_feature_ablation.py

```diff
@@ -345,17 +345,6 @@ def forward_func(inp):
         with self.assertRaises(AssertionError):
             _ = ablation.attribute(inp, perturbations_per_eval=2)
 
-    def test_error_agg_mode_incorrect_fm(self) -> None:
-        def forward_func(inp):
-            return inp[0].unsqueeze(0)
-
-        inp = torch.tensor([[1, 2, 3], [4, 5, 6]])
-        mask = torch.tensor([[0, 1, 2], [0, 0, 1]])
```

Comment:
This test checks when rowwise `mask` is give, the model must not be in "aggregated mode". But I don't think `mask` has anything to do with it. Models of both batch-mode and aggregated-mode may need specify `mask`. So I simply deleted this test.